### PR TITLE
S3 fix and refactoring 

### PR DESCRIPTION
**Pull request by luciferlinx101**
- Write file and image S3 fix
- Refactored s3,resource helper

### CODE REVIEW

File: superagi/helper/resource_helper.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            path = 'input'
+        else:
+            path = 'output'
+
+    print(path + "/" + file_name)
+    resource = Resource(name=file_name, path=path + "/" + file_name, storage_type=storage_type, size=file_size,
+                        type=file_type,
+                        channel="OUTPUT",
+                        agent_id=agent_id)
+    print(resource)
```
Comment (TransformerOptimus) [inline superagi/helper/resource_helper.py:44]: remove this

File: superagi/helper/resource_helper.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+from superagi.config.config import get_config
+from superagi.models.resource import Resource
+import os
+import datetime
+
+
+def make_written_file_resource(file_name: str, agent_id: int, file, channel):
```
Comment (TransformerOptimus) [inline superagi/helper/resource_helper.py:7]: add it in ResourceHelper

**Review by bateller — CHANGES_REQUESTED** [review]
Need to add S3Helper class

File: superagi/helper/s3_helper.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+from superagi.config.config import get_config
+from fastapi import HTTPException, Depends, Request
+
+s3 = boto3.client(
+    's3',
+    aws_access_key_id=get_config("AWS_ACCESS_KEY_ID"),
+    aws_secret_access_key=get_config("AWS_SECRET_ACCESS_KEY"),
+    )
+
+def upload_to_s3(file,path):
```
[the remark this answers is not in the document]
Comment (luciferlinx101) [reply superagi/helper/s3_helper.py]: Fixed

**Pull request merged**